ios/Classes/CameraPermissions.m

```diff
@@ -13,19 +13,7 @@ + (BOOL)checkPermissions {
     NSString *mediaType = AVMediaTypeVideo;
     AVAuthorizationStatus authStatus = [AVCaptureDevice authorizationStatusForMediaType:mediaType];
     
-    __block BOOL permissionsGranted;
-    if (authStatus == AVAuthorizationStatusNotDetermined) {
-        dispatch_semaphore_t sem = dispatch_semaphore_create(0);
-         [AVCaptureDevice requestAccessForMediaType:mediaType completionHandler:^(BOOL granted) {
-           permissionsGranted = granted;
-           dispatch_semaphore_signal(sem);
-         }];
-        dispatch_semaphore_wait(sem, DISPATCH_TIME_FOREVER);
-    } else {
-      permissionsGranted = (authStatus == AVAuthorizationStatusAuthorized);
-    }
-    
-    return permissionsGranted;
```

Comment:
What's the reason for removing this piece of code? I'm not sure what the impact is.

Reply:
This was code I forgot to remove in another PR where I removed the requesting permissions (that functionality should be implemented in the app that uses this lib). The check if the permission exists is still in there.